protocols/virtualdefund/virtualdefund.go

```diff
@@ -17,14 +17,15 @@ import (
 )
 
 const (
-	WaitingForCompleteFinal           protocols.WaitingFor = "WaitingForCompleteFinal"           // Round 1
+	WaitingForFinalStateFromAlice     protocols.WaitingFor = "WaitingForFinalStateFromAlice"
+	WaitingForSignedFinal             protocols.WaitingFor = "WaitingForSignedFinal"             // Round 1
 	WaitingForCompleteLedgerDefunding protocols.WaitingFor = "WaitingForCompleteLedgerDefunding" // Round 2
 	WaitingForNothing                 protocols.WaitingFor = "WaitingForNothing"                 // Finished
 )
 
 const (
-	SignedStatePayload protocols.PayloadType = "SignedStatePayload"
-	InitPayload        protocols.PayloadType = "InitPayload"
+	SignedStatePayload       protocols.PayloadType = "SignedStatePayload"
```

Comment:
We could spit `SignedStatePayload` into two seperate payloads: `SignaturesPayload` and `FinalOutcomePayload`. The `FinalOutcomePayload` could contain the latest voucher amount and be sent out by Alice. The `SignaturesPayload` would be used by all participants to share their signature on the final state.

Reply:
That would be a nice use of the new payload format 👍 Would `FinalOutcomePayload` include Alice's signature? Or would / could she send two payloads at once?

Reply:
I was thinking Alice could send two payloads at once. We might need to tweak some of the message helpers to allow creating a message with more than one payload.

Reply:
I will probably tackle this in a seperate PR.